R-packages/covidcast/R/plot.R

```diff
@@ -346,32 +362,23 @@ plot_choro = function(x, time_value = NULL, include = c(), range,
       package = "covidcast"))
     map_df = map_df %>% dplyr::filter(map_df$LSAD == 'M1') # only get metro and not micropolitan areas
     if (length(include) > 0) {
-      map_df = map_df %>% dplyr::filter(map_df$GEOID %in% include)
+      map_df = map_df %>% dplyr::filter(
+        substr(.$NAME, nchar(.$NAME) - 1, nchar(.$NAME)) %in% include)
     }
     map_df$NAME <- as.character(map_df$NAME)
     map_df = map_df %>% dplyr::mutate(
       is_alaska = substr(NAME, nchar(NAME) - 1, nchar(NAME)) == 'AK',
       is_hawaii = substr(NAME, nchar(NAME) - 1, nchar(NAME)) == 'HI',
       is_pr = substr(NAME, nchar(NAME) - 1, nchar(NAME)) == 'PR',
       color = ifelse(GEOID %in% geo, col_fun(val[GEOID]), missing_col))
-
-    main_df = shift_main(map_df)
-    hawaii_df = shift_hawaii(map_df)
-    alaska_df = shift_alaska(map_df, alaska_centroid)
-    pr_df = shift_pr(map_df, pr_centroid)
-
-    main_col = main_df$color
-    hawaii_col = hawaii_df$color
-    alaska_col = alaska_df$color
-    pr_geo = pr_df$color
   }
 
   else if (attributes(x)$metadata$geo_type == "hrr") {
     map_df = sf::st_read(system.file(
       "shapefiles/hrr/geo_export_ad86cff5-e5ed-432e-9ec2-2ce8732099ee.shp",
       package = "covidcast"))
     if (length(include) > 0) {
-      map_df = map_df %>% filter(.$hrr_num %in% include)
+      map_df = map_df %>% filter(substr(.$hrr_name, 1, 2) %in% include)
```

Comment:
same question

Reply:
First two letters have the state for hrr.

Reply:
Can you add this as a comment here too?